src/autopas/containers/octree/OctreeDirection.h

```diff
@@ -0,0 +1,532 @@
+/**
+ * @file OctreeInnerNode.h
+ *
+ * @author Johannes Spies
+ * @date 18.05.2021
+ */
+
+#pragma once
+
+namespace autopas {
+
+/**
+ * This enum can be used to index the faces of a cube including an "invalid" face.
+ */
+enum Face {
+  O = 0,  // omega/unknown
+  L = 1,
+  R = 2,
+  D = 3,
+  U = 4,
+  B = 5,
+  F = 6,
+};
+
+/**
+ * A datatype that is wide enough to hold faces, edges or vertices.
+ */
+using Any = int unsigned;
```

Comment:
```suggestion
/**
 * A datatype that is wide enough to hold faces, edges or vertices.
 */
using Any = int unsigned;

/**
 * This enum can be used to index the faces of a cube including an "invalid" face.
 */
enum Face : Any {
  O = 0,  // omega/unknown
  L = 1,
  R = 2,
  D = 3,
  U = 4,
  B = 5,
  F = 6,
};
```

That way you make the enum have the underlying type `Any`.